src/pages/nfts/[address].tsx

```diff
@@ -165,6 +176,16 @@ export default function NftByAddress({
   const defaultListing = nft?.listings.find(
     (listing) => listing?.auctionHouse?.address.toString() === HOLAPLEX_MARKETPLACE_ADDRESS
   );
+
+  const otherListings = nft?.listings.filter(
+    (listing) => listing.auctionHouse?.address.toString() !== HOLAPLEX_MARKETPLACE_ADDRESS
+  );
+
+  const magicEdenListing = nft?.listings.find((listing) => listing.auctionHouse === null);
+  const openSeaListing = nft?.listings.find(
+    (listing) => listing?.auctionHouse?.address.toString() === AUCTION_HOUSE_ADDRESSES[1].address
```

Comment:
A magic index is going to get flaky fast. Should assign open sea auction house address to a constant and reference that.